From 813b7230735960f3e5225f1f74ca680c9ea84101 Mon Sep 17 00:00:00 2001
From: Shawn Anastasio <shawn@anastas.io>
Date: Sun, 30 Dec 2018 14:49:43 -0600
Subject: [PATCH] Add powerpc64le as an alias for ppc64

Add powerpc64le as an alias for the ppc64 build target.
---
 src/build-data/arch/ppc64.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/build-data/arch/ppc64.txt b/src/build-data/arch/ppc64.txt
index 9ab7cccded..23d3bb2a10 100644
--- a/src/build-data/arch/ppc64.txt
+++ b/src/build-data/arch/ppc64.txt
@@ -5,6 +5,7 @@ wordsize 64
 
 <aliases>
 powerpc64
+powerpc64le
 ppc64le
 ppc64el
 </aliases>
